src/api/services/initDatasetUri.js

```diff
@@ -1,11 +1,13 @@
 import getDocumentTransformer from '../../common/getDocumentTransformer';
 import { URI_FIELD_NAME } from '../../common/uris';
+import progress from './progress';
+import { INITIALIZING_URI } from '../../common/progressStatus';
 
 const initDatasetUriFactory = ctx =>
     async function initDatasetUri() {
         const count = await ctx.dataset.countWithoutUri();
+        progress.start(INITIALIZING_URI, count);
```

Comment:
Why should'nt you put it before "countWithoutUri" ?

Reply:
To but the count inside the progress as a target

Reply:
My bad --"